src/blockchain/p2p/peer/Address.hpp

```diff
@@ -65,9 +65,9 @@ class Address
 
     Address(pointer address) noexcept;
 
-    virtual ~Address();
+    virtual ~Address() = default;
```

Comment:
This does make a bit of difference, though: now you cannot put a breakpoint on the destructor, which I actually did some time ago chasing a shutdown crash.